src/components/batch-selection/batch-selection.spec.tsx

```diff
@@ -1,18 +1,33 @@
 import React from "react";
-import { mount } from "enzyme";
-import TestRenderer from "react-test-renderer";
+import { mount, ReactWrapper, shallow, ShallowWrapper } from "enzyme";
+
+import BatchSelection, { BatchSelectionProps } from ".";
 import { assertStyleMatch } from "../../__spec_helper__/test-utils";
-import BatchSelection from ".";
+
 import Icon from "../icon";
 import IconButton from "../icon-button";
 import StyledIcon from "../icon/icon.style";
 import StyledIconButton from "../icon-button/icon-button.style";
 
+function renderBatchSelection(
+  props: Omit<BatchSelectionProps, "children">,
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  renderer: any = mount
```

Comment:
annoying that you have to find this `any`, but I can't find another good solution either. I guess we can live with it in a test suite (especially as we're going to move away from Enzyme anyway!). (Even `typeof mount | typeof shallow` doesn't work - TS seems to think you can't call that!)

Reply:
one solution is to just remove the snapshot and not use shallow at all (we already have Chromatic for that). It will be removed anyway with our conversion to RTL.